.ci/docker/Dockerfile

```diff
@@ -43,6 +43,7 @@ WORKDIR /
 
 RUN apt-get remove -y $DEPS_IVERILOG \
  && rm -Rf ghdl-0.36 iverilog-10_3 \
+ && apt-get update \
```

Comment:
Did you need this because of caching?

Reply:
Yes, should I just have thrown away the cache?